lib/Net/Stripe.pm

```diff
@@ -24,6 +25,8 @@ use Net::Stripe::List;
 use Net::Stripe::LineItem;
 use Net::Stripe::Refund;
 
+my $_MAX_API_VERSION = '2017-06-05';
```

Comment:
why not add `$_MIN_API_VERSION` too?

Reply:
conceptually it is there, but it should probably be extracted to the same code location. currently that value is in the subtype, and is hard-coded to 2012-10-26, before which the structure of invoices (or something) is different. but you are correct in that as things progress, there will be a specific max and min for certain code changes.
thx